tests/shell/shell_recorder/listtest.esr

```diff
@@ -16,4 +16,5 @@ RET: 0
 
 < kdb ls $Mountpoint
 
+< kdb rm /sw/kdb/current/plugins
```

Comment:
> Honestly, that sounds like too much effort for a simple test.

Yes, but I think this functionality is also needed elsewhere. Let us discuss further discuss expected behavior in #1483.